.docker/Dockerfile-distroless-static

```diff
@@ -1,4 +1,4 @@
-FROM gcr.io/distroless/static-debian11:nonroot
+FROM gcr.io/distroless/static-debian11:debug-nonroot
```

Comment:
I think this is a big confusing because the "prod" image is now the debug image. Shouldn't this be a dedicated tag?